tests/flow/test_rdbs.py

```diff
@@ -35,6 +35,7 @@ def testRDBCompatibility():
     env = Env()
     env.skipOnCluster()
     skip_on_rlec()
+    env.skipOnAOF()
```

Comment:
why?

Reply:
@danni-m because on AOF redis on start loads the AOF and not the rdb, so as long as between the test the expected data stay the same everything is ok, but now things have changed.